### PR TITLE
Add Tools to a subdomain

### PR DESCRIPTION
**Pull request by Marie59**
Tutorial on how to add tools to a subdomain and build a customize tool panel. This Pr is folowing the creation of subdomain here https://github.com/galaxyproject/training-material/pull/5552

One link will not work in this tuto as it's linked to the tuto in PR above not publish yet

Thank you all ! Looking forward to your reviews !

### CODE REVIEW

**Comment by Marie59** [comment]
Hi all ! Following the recent tuto on creating a subdomain I wrote this one but I don't understand the error... Anyone ?

File: topics/community/tutorials/tools_subdomains/tutorials.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
```
Comment (shiltemann) [inline topics/community/tutorials/tools_subdomains/tutorials.md:1]: this file should be named `tutorial.md` (without the s)

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Add your batch tool to the right section
+
+> <hands-on-title>Tool sections (for batch tools) </hands-on-title>
+>    - Clone the Github repository of the Galaxy server (of where your subdomain to be attached)
+>    - For Galaxy europe clone the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/tree/master)
```
Comment (shiltemann) [inline topics/community/tutorials/tools_subdomains/tutorial.md:43]:
```suggestion
>    - For Galaxy Europe, fork the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/tree/master)
```

File: topics/community/tutorials/tools_subdomains/tutorials.md (hunk 1 above)
Comment (Marie59) [reply topics/community/tutorials/tools_subdomains/tutorials.md]: Oups ...

**Comment by Marie59** [comment]
Thanks @shiltemann for the review !!! (sorry for the stupid error)

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Add your batch tool to the right section
+
+> <hands-on-title>Tool sections (for batch tools) </hands-on-title>
+>    - Clone the Github repository of the Galaxy server (of where your subdomain to be attached)
+>    - For Galaxy Europe, fork the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/tree/master)
```
Comment (bgruening) [inline topics/community/tutorials/tools_subdomains/tutorial.md:43]:
```suggestion
>    - For Galaxy Europe, fork the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools)
```

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+>    - Create a branch on your fork
+>    - Browse through the different yaml files and select the one that correspond to your subdomain for earth system everything is made under the ecology owner.
```
Comment (bgruening) [inline topics/community/tutorials/tools_subdomains/tutorial.md:45]: Which owner? Maybe explain a bit that the name of the files corresponds to a toolshed owner/community and that a few of those communities with a review system and Ci integration gets special trust and auto-updates.

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+{: .agenda}
+
+# Add your batch tool to the right section
+
+> <hands-on-title>Tool sections (for batch tools) </hands-on-title>
+>    - Clone the Github repository of the Galaxy server (of where your subdomain to be attached)
+>    - For Galaxy Europe, fork the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/tree/master)
+>    - Create a branch on your fork
+>    - Browse through the different yaml files and select the one that correspond to your subdomain for earth system everything is made under the ecology owner. 
+>    - Once in the yaml you can add the section for your tool (you want to know what are the existing tool sections ? Go check the [ToolShed's categories](https://toolshed.g2.bx.psu.edu/))
```
Comment (bgruening) [inline topics/community/tutorials/tools_subdomains/tutorial.md:46]: The sections are not from the TS. They are the sections defined in the servers. E.g. here: https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/templates/galaxy/config/tool_conf.xml.j2#L640

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+> ![Image of the modification to make in the ecology.yaml file](../../images/tool_subdomain/ecology_yml_tool.png)
+>
+>   - Then commit your changes and write a nice message for the admin when you open your Pull Request.
+{: .hands_on}
+
+This part is only to make batch tool visible in your subdomain.
+
+# Add a section of tools to your subdomain
+> <hands-on-title>Sections for your subdomain</hands-on-title>
+>    - If not already done clone the Galaxy europe [Infrastructure-playbook repo](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master)
```
Comment (bgruening) [inline topics/community/tutorials/tools_subdomains/tutorial.md:64]:
```suggestion
>    - If not already done clone the Galaxy Europe [Infrastructure-playbook repo](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master)
```

File: topics/community/tutorials/tools_subdomains/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+>
+> ![Image of the modification to make in the global_host_filters.py.j2 file](../../images/tool_subdomain/add_section.png)
+>
+>   - Then commit your changes and write a nice message for the admin when you open your Pull Request.
+{: .hands_on}
+
+
+# Add you interactive tool to the right section
+> <hands-on-title>Interactive tool sections</hands-on-title>
+>    - If not already done clone the Galaxy europe [Infrastructure-playbook repo](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master)
```
Comment (bgruening) [inline topics/community/tutorials/tools_subdomains/tutorial.md:77]:
```suggestion
>    - If not already done clone the Galaxy Europe [Infrastructure-playbook repo](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master)
```

**Review by bgruening — COMMENTED** [review]
Should we also link https://usegalaxy-eu.github.io/operations/subdomains.html ?

File: topics/community/tutorials/tools_subdomains/tutorial.md (hunk 4 above)
Comment (Marie59) [reply topics/community/tutorials/tools_subdomains/tutorial.md]: I just added your sentence here because I don't know more on how this works ;)

**Review by bgruening — APPROVED** [review]

**Pull request merged**